content/doc/administrate/log-management.md

```diff
@@ -231,6 +231,48 @@ clever drain create NewRelicHTTP "https://log-api.eu.newrelic.com/log/v1" --api-
 NewRelic has two zones, **EU** and **US**. An account on one zone is not available on the other, make sure to target the right intake endpoint (`log-api.eu.newrelic.com` or `log-api.newrelic.com`).
 {{< /callout >}}
 
+### OVH Logs Data Platform
+
+To export logs from an application or an add-on to [OVH Logs Data Platform](https://help.ovhcloud.com/csm/en-ie-logs-data-platform-quick-start?id=kb_article_view&sysparm_article=KB0055819), use the following setup:
+
+- A **TCP** drain log with `clever drain create TCPSyslog`
+- Your Logs Data Platform **host** with **port** `514` (SSL ports aren't supported for TCP drains)
+- The **write token** for your stream (provided on your Logs Data Platform console)
+
+On your terminal, use the following command:
+
+{{< tabs items="Application,Add-on" >}}
+
+  {{< tab >}}**Exporting logs from an application**:
+    
+  ```shell
+  clever drain create TCPSyslog tcp://<host>:514 -a <application-alias> --sd-params="X-OVH-TOKEN=\"<token>\""
+  ```
```

Comment:
This is for a linked application. Eventually, we could add the alternative that will work on any application without having the link it locally:

```shell
  clever drain create TCPSyslog tcp://<host>:514 -app <application-id-or-name> --sd-params="X-OVH-TOKEN=\"<token>\""
  ```

Note that this alternative has been introduced in version 3.8.0 and applies to many commands (see https://github.com/CleverCloud/clever-tools/issues/587)

Reply:
I like it better, yes. Thank you!

Reply:
```suggestion
  ```shell
  clever drain create TCPSyslog tcp://<host>:514 -app <application-id-or-name> --sd-params="X-OVH-TOKEN=\"<token>\""
  ```
```